src/com/rcv/CVRReader.java

```diff
@@ -26,6 +26,29 @@
   // container for all CastVoteRecords parsed from the input file
   public final List<CastVoteRecord> castVoteRecords = new ArrayList<>();
 
+  // purpose: helper function to wrap file IO with error handling
```

Comment:
What's the rationale for moving this method definition higher up in the file? I think the style guidelines just say you should have some non-arbitrary ordering scheme, which we haven't really been bothering to do... but I've generally been putting public higher than private (since that's what I'm used to from Rails). I'm fine with a different scheme (private before public?), though.